README.md

```diff
@@ -13,6 +13,20 @@ compatible with stylesheets designed for [Pygments][].
 
 [Pygments]: http://pygments.org "Pygments"
 
+## Installation
+
+In your Gemfile, add:
+
+```ruby
+gem 'rouge', '~> 3.26.0'
```

Comment:
Please remove the version string. It's probably best to leave it off since it will become outdated. Most people can use the latest version without problem, and those that don't want to do that probably know how to pin.